app/views/videos/show.html.haml

```diff
@@ -10,4 +10,33 @@
         %p= @video.description
         .actions
           %a.btn.btn-primary(href="") Watch Now
-          %a.btn.btn-default(href="") + My Queue
\ No newline at end of file
+          %a.btn.btn-default(href="") + My Queue
+
+%section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_for [@video, Review.new] do |f|
+        %fieldset
+          .form-group
+            %label Rate this video
+            .row
+              .col-sm-3
+                = f.select :rating, options_for_select([["5 Stars", 5], ["4 Stars", 4], ["3 Stars", 3], ["2 Stars", 2], ["1 Star", 1]])
+            %label Write Review
+            = f.text_area :content, rows: 6, class: "span6"
+            .row
+              .col-sm-8
```

Comment:
You can do the label inilne `, label: "Write Review"` Also the `"span6"` is a part of an older template. The template is slightly updated from the solution video and you'll want to make your code here match that of the template.  The lines that need to change here are 25-28. Everything else can stay the same.